plugins/cros_ec-usb/fu-cros_ec-usb-common.c

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2020 Benson Leung <bleung@chromium.org>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include <string.h>
+
+#include "fu-cros_ec-usb-common.h"
+
+gboolean
+fu_cros_ec_usb_parse_version (const gchar *version_raw,
+			      struct cros_ec_version *version)
```

Comment:
I think when you add self tests there should probably be one that does parsing of this to match all your boundary conditions properly.